release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2707,6 +2707,11 @@ In the following tables, features are marked with the following statuses:
 |Technology Preview
 |General Availability
 
+|Hosted control planes for {product-title} on ARM
```

Comment:
```suggestion
|Hosted control planes for an ARM64 {product-title} cluster on AWS
```

Reply:
Thank you @lahinson 
I have updated the PR as per above. Thanks.